app/src/main/java/com/example/themoviedb/SplashScreenActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.themoviedb
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import java.lang.Exception
+
+class SplashScreenActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        val background = object : Thread() {
```

Comment:
Would be better to use handler postDelayed than thread.sleep() . Sleep simply blocks the thread (in this case the Main Thread) which is smething we don't want to do. https://stackoverflow.com/questions/18712992/handler-postdelayed-and-thread-sleep